client/main.lua

```diff
@@ -76,62 +71,65 @@ local function areKeysJobShared(veh)
     for job, v in pairs(config.sharedKeys) do
         if job == QBX.PlayerData.job.name then
             if config.sharedKeys[job].requireOnduty and not QBX.PlayerData.job.onduty then return false end
+
             for _, vehicle in ipairs(v.vehicles) do
                 if string.upper(vehicle) == string.upper(vehName) then
                     if not hasKeys(vehPlate) then
-                        TriggerServerEvent("qb-vehiclekeys:server:AcquireVehicleKeys", vehPlate)
+                        TriggerServerEvent('qb-vehiclekeys:server:AcquireVehicleKeys', vehPlate)
                     end
+
                     return true
                 end
             end
         end
     end
+
     return false
 end
 
-local function setVehicleDoorLock(veh, state, anim)
-    if veh then
-        if not isBlacklistedVehicle(veh) then
-            if hasKeys(qbx.getVehiclePlate(veh)) or areKeysJobShared(veh) then
-                local vehLockStatus = GetVehicleDoorLockStatus(veh)
+---manages the opening of locks
+---@param vehicle number? The entity number of the vehicle.
+---@param state boolean? State of the vehicle lock.
+---@param anim any Aniation
+local function setVehicleDoorLock(vehicle, state, anim)
+    if not vehicle then return end
+    if not isBlacklistedVehicle(vehicle) then
+        if hasKeys(qbx.getVehiclePlate(vehicle)) or areKeysJobShared(vehicle) then
+
```

Comment:
blank space can go

Reply:
Improves readability by separating animation from the rest of the context